.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -1,8 +1,6 @@
 name: Optimize Release Camunda Optimize C8
 
 on:
-  schedule:
```

Comment:
I dont know if we still should do this

Reply:
I think this is for smoke testing the workflow on a schedule. 
I do not mind removing it if it is necessary.

Reply:
It's not necesary to remove it, I was just thinking if this isn't something c7 related

Reply:
i brought it back